deployments/gpu-operator/templates/clusterpolicy.yaml

```diff
@@ -511,8 +511,9 @@ spec:
     {{- if .Values.dcgmExporter.args }}
     args: {{ toYaml .Values.dcgmExporter.args | nindent 6 }}
     {{- end }}
-    {{- if .Values.dcgmExporter.config }}
-    config: {{ toYaml .Values.dcgmExporter.config | nindent 6 }}
+    {{- if ((.Values.dcgmExporter.config).name) }}
```

Comment:
```suggestion
    {{- if  and (.Values.dcgmExporter.config) (.Values.dcgmExporter.config.name) }}
```

Reply:
What I've written is basically the exact same effect but simpler. Do you still want this change?

Reply:
Oh, is `if ((.Values.dcgmExporter.config).name)` is shorthand equivalent of the AND check I suggested above?

Reply:
Yes

Reply:
@chipzoller can you point us to helm documentation that covers this shortened syntax? I would prefer Tariq's suggestion for better readability.

Reply:
I don't know it's specific to Helm; it's just general Go templating and something I've learned over years of working with the tool. But I just changed it according to your suggestion.

Reply:
Maybe I am having an issue viewing this in GitHub, but I do not see the change applied. I still see the shortened syntax:
```
{{- if ((.Values.dcgmExporter.config).name) }}
```

Reply:
Do you not see it [here](https://github.com/NVIDIA/gpu-operator/pull/949/commits/ba4c9e1f1a6594d6598d797c7589f778410dbf99)?

Reply:
The commit you linked is updating the wrong file.

Reply:
🤦 so sorry. Changed the wrong template. Should be updated.